foca/errors/errors.py

```diff
@@ -7,7 +7,8 @@
 from connexion.exceptions import (
     ExtraParameterProblem,
     Forbidden,
-    Unauthorized
+    Unauthorized,
+    InvalidSpecification
```

Comment:
Generally, please always add a `,` even to the last item, when you enumerate objects in list or similar. This makes it easier to maintain in case values are re-arranged, added to or otherwise moved around.